eng/build.yml

```diff
@@ -34,3 +24,11 @@ steps:
     command: test
     arguments: -c $(BuildConfiguration) --no-build /p:CollectCoverage=true /p:CoverletOutputFormat=opencover /p:Include="[coverlet.collector]*%2c[coverlet.core]*%2c[coverlet.msbuild.tasks]*" /p:Exclude="[coverlet.core.tests.samples.netstandard]*%2c[coverlet.tests.xunit.extensions]*"
     testRunTitle: $(Agent.JobName)
+
+- template: publish-coverlet-result-files.yml
```

Comment:
I really like that we collect logs here etc... Will make it easier to troubleshoot issues that only happen on the build server. ❤️ 👍